Tutorials/Porting-Packages-V8/index.md

```diff
@@ -0,0 +1,178 @@
+                            : string.Join("", val.StripHtml().StripNewLines().Take(200)));
+                    }
+                }
+            }
+        }
+    }
+
+
+
+### New Composer & Component way
+
+    using System.Linq;
+    using Umbraco.Core;
+    using Umbraco.Core.Components;
+    using Umbraco.Core.Events;
+    using Umbraco.Core.Models;
+    using Umbraco.Core.Services;
+    using Umbraco.Core.Services.Implement;
+
+    namespace MyProject.Components
+    {
+        [RuntimeLevel(MinLevel = RuntimeLevel.Run)]
+        public class MyComposer : IUserComposer
+        {
+            public void Compose(Composition composition)
+            {
+                composition.SetContentLastChanceFinder<My404ContentFinder>();
+
+                composition.Components().Append<MyComponent>();
+            }
+        }
+
+        public class MyComponent : IComponent
+        {
+            public void Initialize()
+            {
+                ContentService.Saving += this.ContentService_Saving;
+            }
+
+            public void Terminate()
+            {
+            }
+
+            /// <summary>
+            /// Listen for when content is being saved, check if it is a
+            /// new item and fill in some default data.
+            /// </summary>
+            private void ContentService_Saving(IContentService sender, SaveEventArgs<IContent> e)
+            {
+                foreach (var content in e.SavedEntities
+                    //Check if the content item type has a specific alias
+                    .Where(c => c.ContentType.Alias.InvariantEquals("MyContentType"))
+                    //Check if it is a new item
+                    .Where(c => c.HasIdentity == false))
+                {
+                    //check if the item has a property called 'richText'
+                    if (content.HasProperty("richText"))
+                    {
+                        //get the rich text value
+                        var val = content.GetValue<string>("richText");
+
+                        //if there is a rich text value, set a default value in a
+                        // field called 'excerpt' that is the first
+                        // 200 characters of the rich text value
+                        content.SetValue("excerpt", val == null
+                            ? string.Empty
+                            : string.Join(string.Empty, val.StripHtml().StripNewLines().Take(200)));
+                    }
+                }
+            }
+        }
+    }
+
+
+## Update package.xml for Umbraco ZIP packages
+Currently the Umbraco packager in the backoffice is still undergoing changes (at the time of writing) and are unable to use the friendly packager UI in Umbraco.
+
+As version 7 and version 8 of Umbraco has breaking changes in its APIs and code, its recommended to ship a different version of your package to support V8 only. So update the package.xml in your Umbraco package zip to have
```

Comment:
Seems like half of the sentence is missing here? 🤔

Reply:
@jmayntzhusen & @Shazwazza I have updated the missing copy & example XML from this section.